great-idea/css/index.css

```diff
@@ -63,4 +63,356 @@ h1, h2, h3, h4, h5 {
+	border-spacing: 0;
+}
+
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+		width: 875px;
+		margin: 0 auto;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+nav {
+	width: 875px;
+	padding-top: 20px;
+	display: flex;
+	justify-content: space-between;
+}
+
+nav .logo {
+	height: 40px;
+	margin: 0;
+	padding: 0;
+	vertical-align: middle;
+}
+
+nav > img {
+	margin-left: 200px;
+}
+
+nav > a {
+	font-family: 'Titillium Web', sans-serif;
+	color: silver;
+	text-decoration: none;
+	margin-right: 50px;
+	margin-top: 15px;
+}
+
+.nav-logo {
+	margin-top: 0;
+}
+
+header {
+	height: 450px;
+	margin-bottom: 50px;
+	margin-top: 50px;
+	border-bottom: 2px solid black;
+	display: flex;
+}
+
+h1 {
+	font-size: 80px;
+	display: inline-block;
+	padding-bottom: 20px;
+	text-align: center;
+}
+
+.head-img {
+	display: inline-block;
+	padding-left: 70px;
+	margin-bottom: 30px;
+}
+
+.head-contact {
+	font-size: 40px;
+	margin-top: 50px;
+	margin-bottom: 30px;
+	text-align: center;
+}
+
+.contactimg {
+	width: 875px;
+	height: 250px;
+	margin-right: 15px;
+	margin-top: 30px;
+}
+
+.derp {
+	display: flex;
+	justify-content: space-around;
+	flex-direction: column;
+}
+
+.head1 {
+	padding-top: 20px;
+	padding-left: 50px;
+	padding-right: 50px;
+	margin-top: 80px;
+}
+
+.derp > a {
+	text-decoration: none;
+	border: 1px solid black;
+	color: black;
+	padding: 10px 30px;
+	margin: 0 0 130px 140px;
+	text-align: center;
+	width: 150px;
+}
+
+.service {
+	border-bottom: 2px solid black;
+	display: flex;
+	flex-direction: column;
+	height: 370px;
+}
+
+.services-header-img {
+	height: 200px;
+	width: 100%;
+}
+
+.services {
+	width: 875px;
+	margin-top: 30px;
+}
+
+.services > h2 {
+	font-size: 30px;
+}
+
+section {
+	display: flex;
+}
+
+.types {
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.content1 p {
+	width: 410px;
+	margin-bottom: 20px;
+}
+
+.content2 p {
+	width: 275px;
+	margin-bottom: 20px;
+}
+
+.derpderp {
+	height: 220px;
+	width: 400px;
+	border: 1px solid black;
+	background-color: #EAECEE;
+	padding: 15px;
+	margin: 15px;
+}
+
+.derpderp:last-of-type {
+	margin-bottom: 40px;
+}
+
+button {
+	height: 35px;
+	width: 120px;
+	border: 1px solid black;
+	border-radius: 10px;
+	margin-top: 20px;
+}
+
+.middle-img {
+	margin-top: 40px;
+	margin-bottom: 40px;
+}
+
+.facts {
+	display: flex;
+	flex-direction: row;
+	border-top: 2px solid black;
+	padding-top: 40px;
+	margin-bottom: 40px;
+}
+
+.text {
+	padding: 0 15px;
+}
+
+.text > ul {
+	line-height: 2;
+	padding-top: 20px;
+	padding-left: 15px;
+	list-style-type: square;
+}
+
+form {
+	line-height: 1.5;
+	width: 600px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	border: 2px dashed black;
+	margin-left: 150px;
+	margin-bottom: 40px;
+	padding: 50px;
+	background-color: #EEEAEA
+}
+
```

Comment:
```suggestion

input {
	font-size: 20px;
	padding: 10px;
	line-height: 20px;
}
```

Reply:
Excellent job on the contact form! Your project looks excellent. Keep reaching for those stretch challenges. They hide some awesome easter eggs in the stretch challenges that will make you a strong developer quickly. If you want to make those input fields a bit bigger, here's some suggested styles. Other than that, your project looks fantastic.